tests/integration/targets/docker_plugin/tasks/main.yaml

```
@@ -0,0 +1,24 @@
+- name: Create random name prefix
+  set_fact:
+    name_prefix: "{{ 'ansible-test-%0x' % ((2**32) | random) }}"
+    plugin_names: []
+
+- debug:
+    msg: "Using name prefix {{ name_prefix }}"
+
+- block:
+  - include_tasks: run-Itest.yml
```

```suggestion
  - include_tasks: run-test.yml
```
Also rename the file from `run-Itest.yml` to `run-test.yml`.